.gitignore

```diff
@@ -3,31 +3,20 @@
 .ipynb_checkpoints
 dist/
 /wandb/
-.wandb_core/
 __pycache__
-artifacts
 .idea
-local-artifacts
-weave_internal.egg-info
-weave.egg-info/
-weave_query/weave_query/frontend/assets
+*.egg-info/
 build/
 MNIST
 food-101
 food-101.tar.gz
 gha-creds-*.json
-.vscode/launch.json
+.vscode
 .mypy_cache
 .hypothesis
 .env
-yarn-error.log*
-config/.admin.env
-config/.wb_api.env
-weave/tests/zlib.txt
```

Comment:
so much nicer

Reply:
but just to be clear, this is gone?

Reply:
what if i already have an admin.env file in that folder, will this now appear in git?

Reply:
We don't have a `config` dir anymore -- do you need it?